src/lib/services/elementPositionService.js

```diff
@@ -0,0 +1,24 @@
+const isBrowser = typeof window !== 'undefined';
+
+export const getElementsPositionDifference = (contentEl, viewportEl) => {
+  if (!isBrowser) {
```

Comment:
I assume this is here for tests. I think a cleaner solution would be to just mock this service.